src/qt/rpcconsole.cpp

```diff
@@ -845,7 +845,7 @@ void RPCConsole::clearSelectedNode()
 {
     ui->peerWidget->selectionModel()->clearSelection();
     cachedNodeids.clear();
-    ui->detailWidget->hide();
+    ui->widget_2->hide();
```

Comment:
The details sidebar collapses completely now when disconnecting or banning a node. It's not a big deal, but I thought it was worth mentioning since it appears by default when initially switching to the tab in the dialog. To make the interaction more uniform, we could:

 a. Hide the sidebar by default and leave this code as-is to show it when selecting a node
 b. Delete `show()` and `hide()` calls on `widget_2` so the sidebar is always visible

If we go with _a_, we can delete the "Select a peer..." label and its visibility toggling--no one will ever see it. It may be worth renaming `widget_2` with the name of the old `detailWidget` so the code is more clear--there's a lot stuffed into the rpcconsole form.

Reply:
For the love of all that is holy please, please use the name `detailWidget`.

Reply:
widget_1 and widget_2 were preexisting objects subordinate to the splitter at the level above detailWidget. I will rename both. There are many objects that are still default names that were assigned by the designer when the screen was originally built.

Reply:
Done